app/views/landing_page/blocks/_columns_layout.html.erb

```diff
@@ -1,5 +1,8 @@
-<div class="govuk-grid-row">
-  <% block.columns.each do |column| %>
-    <div class="<%= column_class_for_equal_columns(block.columns.count) %>"><%= render "landing_page/blocks/#{column.type}", block: column %></div>
+<%
+  add_view_stylesheet("landing_page/columns_layout")
+%>
+<div class="app-b-columns-layout" data-column-count="<%= block.data["column_count"] || 3 %>">
```

Comment:
I'd consider renaming `column_count`. I thought it meant the total number of columns, and that the number of configured columns had to match the total. 

Perhaps it should be called `number_of_columns_per_row`?

Reply:
The key point here is that we separate columns from blocks - so this _is_ a column count. You don't have configured columns, you configure blocks that are layed out into columns.

Reply:
I can see both perspectives :) Another 'naming things' problem! I'm personally OK with `column_count` but can see that it's potentially confusing.  What about just `columns` or is that even more ambiguous?

Reply:
`number_of_columns`?

Reply:
I'd prefer to keep these shorter if possible. Easier to read. What's wrong with just `columns`?

Reply:
I think `columns` is fine, so unless anyone has an objection, I'll go with that.